tools/stringtable_validator.py

```diff
@@ -145,24 +145,26 @@ def main():
     print("Validating Stringtables")
     print("-----------------------")
 
-    # Allow running from root directory and tools directory
```

Comment:
I think the changes to `stringtable_validator.py` are not needed since it already checked all `stringtable.xml` files in the project.

Reply:
It is not required but if someone where to add a stringtable.xml into includes folder it could start complaining. Ideally we should merge the folder selection logic into some helper script.

Reply:
Yeah, I see how that could be a problem. Would be fine to re-revert the changes (sorry 😄) or keep as is then.